internal/core/layout.rs

```diff
@@ -593,6 +593,62 @@ pub fn organize_grid_layout(input_data: Slice<GridLayoutInputData>) -> GridLayou
     organized_data
 }
 
+// The finalizer inserts the pos and size into the result array (which becomes the layout cache property),
+// including the indirections for repeated items (so that the x,y,width,height properties for repeated items
+// can point to indices known at compile time, those that contain the indirections)
+// Example: for repeater_indices=[1,4] (meaning that item at index 1 is repeated 4 times),
+// result=[0.0, 80.0, 4.0, 5.0, 80.0, 80.0, 160.0, 80.0, 240.0, 80.0, 320.0, 80.0]
+//  i.e. pos1, width1, jump to idx 4, jump to idx 5, pos2, width2, pos3, width3, pos4, width4, pos5, width5
```

Comment:
You can use doc comments here with `///`
(and also for the fields)
Then it shows up in the docs with `--document-private-items`